app/jobs/check_catalogues_job.rb

```diff
@@ -43,15 +40,15 @@ def catalogue_gather
         set_catalogue_region(region)
 
         #scrape the catalogue number information
-        catalogue_result += scrape_catalogue_number()
+        published_catalogue_nums += scrape_catalogue_number
         
         sleep session_wait
       end
     end
-    return catalogue_result
+    return published_catalogue_nums
   end
 
-  def scrape_catalogue_number()
+  def scrape_catalogue_number
     catalogue_number = []

```

Comment:
Consider renaming the method so its clear that its dealing with a collection of catalogue numbers. The singular naming `scrape_catalogue_number` implies that the method returns one number only.

Consider renaming:
- method: `def scrape_catalogue_number` to `def scrape_catalogue_numbers`
- variable (line 52): `catalogue_number` to `catalogue_nums`